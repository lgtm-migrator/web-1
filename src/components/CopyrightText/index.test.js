import React from "react"
import renderer from "react-test-renderer"
import { CopyrightText } from "."

describe("Header", () => {
  it("renders correctly with site title", () => {
    const tree = renderer
      .create(<CopyrightText siteTitle="Test site name" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
