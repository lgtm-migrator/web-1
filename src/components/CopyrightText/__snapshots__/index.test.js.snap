// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with site title 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
>
  <small>
    Â© 
    2020
     
    <strong>
      Test site name
    </strong>
  </small>
</p>
`;
